src/getExtensionApi.ts

```diff
@@ -33,10 +34,15 @@ export async function getGitApi(): Promise<API> {
             return gitExtension.getAPI(1);
         }
```

Comment:
nit: To make it more clear what the error on line 45 is for, I think adding an else statement and throwing the error in the else makes it easier to understand. Easier to understand that the error is only for when `gitExtension` is undefined.

```suggestion
        } else {
         	throw new Error(localize('unableGit', 'Unable to retrieve VS Code Git API. Please ensure git is properly installed and reload VS Code.'));
        }
```

Reply:
Haha, that did not work the way I wanted it to 😬